scalikejdbc-mapper-generator/src/sbt-test/scalikejdbc-mapper-generator/gen/src/test/scala/ProgrammersTest.scala

```diff
@@ -38,6 +38,9 @@ class ProgrammersTest extends fixture.FlatSpec with Matchers with AutoRollback {
     programmers.foreach(Programmers.destroy(_))
     Programmers.findAll().toSet should equal(Set.empty[Programmers])
     Programmers.countAll() should equal(0)
+
+    val res = Programmers.batchInsert(programmers.toList)
```

Comment:
@seratch

To remove `toList` I changed, around on line 18's, from 'Set' to 'Seq' of `val programmers`,
and applied `toSet` for results of `findAll` method, also removed `toList` from a Random.shuffle argment `programmers`.